store/bolt/user_store_test.go

```diff
@@ -0,0 +1,112 @@
+package bolt_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/gopheracademy/congo"
+	"github.com/gopheracademy/congo/store/bolt"
+)
+
+// Ensure the store can create a user.
+func TestUserStore_CreateUser(t *testing.T) {
+	s := OpenStore()
+	defer s.Close()
```

Comment:
These two lines open our test wrapper, `bolt_test.Store`, in a temporary file path. The `defer` close will close our store and delete it from the temp path automatically.